### DISCUSSION TITLE
Does 2.3.x milvus supports connection pooling? If yes then, how can it be used in python using pymilvus? Can you please provide a example?

### DISCUSSION THREAD

**Discussion by vishal549** [milvus-io/milvus#27052] [Q&A and General discussion]
I have a use case where multiple REST APIs can use milvus DB concurrently. For this how can I share the connection?

**Comment by yhmo** [comment]
We don't intend to implement a connection pool for each sdk, because it is difficult to maintain(For different sdk, the implementation is different).

If your app is written by python(FastAPI, Flask, etc), then it is thread-safe to share a connection across threads. Create a connection for each thread is also ok.

**Reply by xiaofan-luan** [reply]
I think someone is working on that already
